File: CampusServer/src/main/java/com/cai/campus/server/entity/SignIn.java
package com.cai.campus.server.entity;

import java.io.Serializable;

/**
 * (SignIn)实体类
 *
 * @author makejava
 * @since 2020-06-01 16:09:24
 */
public class SignIn implements Serializable {
    private static final long serialVersionUID = -62269484241224203L;

    private Integer signId;

    private Integer groupId;

    private Long createTime;

    private Long endTime;

    private Integer uid;

    private String location;

    private String detail;

    public SignIn() {
    }

    public SignIn(Integer groupId) {
        this.groupId = groupId;
    }

    public Integer getSignId() {
        return signId;
    }

    public void setSignId(Integer signId) {
        this.signId = signId;
    }

    public Integer getGroupId() {
        return groupId;
    }

    public void setGroupId(Integer groupId) {
        this.groupId = groupId;
    }

    public Long getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Long createTime) {
        this.createTime = createTime;
    }

    public Long getEndTime() {
        return endTime;
    }

    public void setEndTime(Long endTime) {
        this.endTime = endTime;
    }

    public Integer getUid() {
        return uid;
    }

    public void setUid(Integer uid) {
        this.uid = uid;
    }

    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public String getDetail() {
        return detail;
    }

    public void setDetail(String detail) {
        this.detail = detail;
    }

}

File: CampusClient/app/src/main/java/com/cai/campus/common/network/model/SignIn.kt
package com.cai.campus.common.network.model

import java.io.Serializable

/**
 * (SignIn)实体类
 *
 * @author makejava
 * @since 2020-06-01 16:09:24
 */
class SignIn : Serializable {
    var signId: Int? = null
    var groupId: Int? = null
    var createTime: Long? = null
    var endTime: Long? = null
    var uid: Int? = null
    var location: String? = null
    var detail: String? = null

    constructor() {}
    constructor(groupId: Int?) {
        this.groupId = groupId
    }

    companion object {
        private const val serialVersionUID = -62269484241224203L
    }

    override fun toString(): String {
        return "SignIn(signId=$signId, groupId=$groupId, createTime=$createTime, endTime=$endTime, uid=$uid, location=$location, detail=$detail)"
    }


}